zipline/assets/futures.py

```diff
@@ -0,0 +1,128 @@
+from pandas import Timedelta
+
+from zipline.finance.trading import TradingEnvironment
+
+
+class FutureChain(object):
+    """ Allows users to look up future contracts.
+
+    Parameters
+    ----------
+    trading_algorithm : TradingAlgorithm
+    root_symbol : str, optional
+    as_of_date : pandas.Timestamp
+    """
+
+    def __init__(self, trading_algorithm, root_symbol='', as_of_date=None):
```

Comment:
It should be a kwarg, as it is, but I don't think it should have a default value. Failing to pass a root_symbol should be a failure case.